src/MCMCChains.jl

```diff
@@ -17,10 +18,10 @@ import Tables
 import TableTraits
 import IteratorInterfaceExtensions
 
-using LinearAlgebra: diag
+using LinearAlgebra # : diag, BlasReal, dot
```

Comment:
Would
```suggestion
using LinearAlgebra: diag, dot
```
be sufficient? I think it's nice if we can clearly state which functions we actually use from each package.

Reply:
I wrote
```suggestion
using LinearAlgebra : diag, BlasReal, dot
```
and used `BlasReal` instead of `LinearAlgebra.BlasReal` in `src/ess.jl`

Reply:
Ah I see :+1: